osu.Game/Overlays/FullscreenOverlay.cs

```diff
@@ -27,37 +29,58 @@ public abstract class FullscreenOverlay<T> : WaveOverlayContainer, INamedOverlay
         [Cached]
         protected readonly OverlayColourProvider ColourProvider;
 
+        [Cached]
+        protected OngoingOperationTracker OngoingOperationTracker { get; private set; }
+
+        private readonly LoadingLayer loadingLayer;
+
         protected FullscreenOverlay(OverlayColourScheme colourScheme, T header)
         {
             Header = header;
-
             ColourProvider = new OverlayColourProvider(colourScheme);
 
+            AddRangeInternal(new Drawable[]
```

Comment:
This shouldn't be adding to the internal content, you'll make the loading layer not move with the entire content when hiding the overlay, as can be seen in the video you attached.

Reply:
Yeah, just wasn't sure if it's okay or not, so I decided to not touch it until feedback

Reply:
As peppy mentioned this in not what we really want, but fixed this exact case: https://streamable.com/oej6kt